diff --git a/ActionListItem.qml b/cActionListItem.qml
index 0708453..9198fc2 100644
--- a/ActionListItem.qml
+++ b/cActionListItem.qml
@@ -24,55 +24,60 @@ import org.kde.pim.mobileui 4.5 as KPIM
 
 
 QML.Item {
-  id : _top
-  height : parent.height
-  width : (parent ? parent.width : 0)
-  property string name
-  property string argument : ""
-  property string title : ""
-  property bool reactsOnLongPressed : false
-
-  property string category
-
-  signal triggered(string triggeredName)
-  signal doCollapse()
-  signal pressAndHold()
-
-  onVisibleChanged :
-  {
-    if (!visible)
-      height = -actionItemSpacing
-    else
-      height = actionItemHeight
-  }
-
-  KPIM.Action {
-    height : parent.height
-    width : parent.width
-    showIcon : false
-    action : {
-      application.setActionTitle(name, title)
-      application.getAction(name, argument);
-    }
-    actionIconName : {
-      application.getActionIconName(name);
-    }
-    hidable : false
+    id: _top
+    height: parent.height
+    width: (parent ? parent.width : 0)
 
-    onLongPressed: {
-      pressAndHold();
-    }
+    property string name
+    property string argument: ""
+    property string title: ""
+    property bool reactsOnLongPressed: false
+    property string category
+
+    signal triggered(string triggeredName)
+    signal doCollapse()
+    signal pressAndHold()
+
+    onVisibleChanged: {
 
-    onTriggered : {
-      parent.doCollapse()
-      parent.triggered(name)
+        if (!visible) {
+            height = -actionItemSpacing
+        } else {
+              height = actionItemHeight
+        }
     }
 
-    QML.Image {
-      anchors.right: parent.right
-      anchors.verticalCenter: parent.verticalCenter
-      source: KDE.locate( "data", "mobileui/long-press-indicator.png" )
-      visible: _top.reactsOnLongPressed
+    KPIM.Action {
+        height: parent.height
+        width: parent.width
+        showIcon: false
+        hidable: false
+
+        action: {
+            application.setActionTitle(name, title);
+            application.getAction(name, argument);
+        }
+
+        actionIconName: {
+            application.getActionIconName(name);
+        }
+
+        onLongPressed: {
+            pressAndHold();
+        }
+
+        onTriggered: {
+            parent.doCollapse()
+            parent.triggered(name)
+        }
+
+        QML.Image {
+            anchors {
+                right: parent.right
+                verticalCenter: parent.verticalCenter
+            }
+            source: KDE.locate( "data", "mobileui/long-press-indicator.png" )
+            visible: _top.reactsOnLongPressed
+        }
     }
-  }
 }
