diff --git a/ActionButton.qml b/cActionButton.qml
index 0cb7e6b..ba5ba1c 100644
--- a/ActionButton.qml
+++ b/cActionButton.qml
@@ -26,25 +26,29 @@ import org.kde.pim.mobileui 4.5 as KPIM
  * The button will update its enabled state according to the
  * associated action.
  */
+
 KPIM.Button {
-  property variant action
-  property string actionName
 
-  width: 70
-  height: 70
-  action: application.getAction( actionName, "" )
-  enabled: action.enabled
-  opacity: enabled ? 1 : 0.65
+    property variant action
+    property string actionName
+
+    width: 70
+    height: 70
+    action: application.getAction( actionName, "" )
+    enabled: action.enabled
+    opacity: enabled ? 1 : 0.65
 
-  onClicked : { action.trigger() }
+    onClicked: { 
+        action.trigger() 
+    }
 
-  Connections {
-    target: action
+    Connections {
+        target: action
 
-    onChanged: {
-      border.width = action.checked ? 2 : 0
+        onChanged: {
+            border.width = action.checked ? 2 : 0
+        }
     }
-  }
 
-  border.width : action.checked ? 2 : 0
+    border.width: action.checked ? 2 : 0
 }
